polyfills/Array/prototype/fill/polyfill.js

```diff
@@ -0,0 +1,29 @@
+Object.defineProperty(Array.prototype, 'fill', {
+	configurable: true,
+	value: function fill(value) {
+		if (this === undefined || this === null) {
+			throw new TypeError(this + ' is not an object');
+		}
+
+		var arrayLike = this instanceof String ? this.split('') : Object(this);
+
+		var length = Math.max(Math.min(arrayLike.length, 9007199254740991), 0) || 0;
```

Comment:
Maybe this should rely on `Number.MAX_SAFE_INTEGER` (or a polyfilled version of it) for clarity? Can polyfills rely on other polyfills?